### PR TITLE
Added option to avoid C style. #354

### PR DESCRIPTION
**Pull request by maxvetrenko**

### CODE REVIEW

File: src/xdocs/config_misc.xml [hunk 1]
```diff
@@ -852,6 +852,12 @@ messages.properties: Key 'ok' missing.
             <td><a href="property_types.html#integer">Integer</a></td>
             <td>4</td>
           </tr>
+          <tr>
+            <td>forceStrictCondition</td>
+            <td>force strict condition in line wrapping case.</td>
```
Comment (mkordas) [inline src/xdocs/config_misc.xml]: please add better description and some samples to documentation

**Comment by romani** [comment]
@maxvetrenko ,

1) 
> + if (!mForceStrictCondition && aCurrentNode.getColumnNo() < aCurrentIndent
+ || mForceStrictCondition && aCurrentNode.getColumnNo() != aCurrentIndent)

Please move mForceStrictCondition to separate IF. 

2) 
please follow advice from @mkordas

**Comment by maxvetrenko** [comment]
@romani,
1) >> Please move mForceStrictCondition to separate IF.

Done: https://github.com/maxvetrenko/checkstyle/blob/18b7eefcbed275aa01c914d6a8be4ab13e2f82a3/src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/LineWrappingHandler.java#L310

2) >> please follow advice from @mkordas

Tried to fixed it. Sorry for deleting his comment, but I had to rewrite hole commit.

**Comment by romani** [comment]
merged as FF

**Pull request closed without merging**